lib/unicorn_logstash_patch.rb

```diff
@@ -5,10 +5,16 @@
 # However, that means we get a separate logstash message for each backtrace which isn't what we want. The
 # monkey patch here overrides Unicorn's logging of error so that we log the error and backtrace in a
 # single message.
-module Unicorn
-  def self.log_error(logger, prefix, exc)
-    message = exc.message
-    message = message.dump if /[[:cntrl:]]/ =~ message
-    logger.error "#{prefix}: #{message} (#{exc.class})\n#{exc.backtrace.join("\n")}"
+module UnicornLogstashPatch
```

Comment:
Ahh I added this previously but should we move this to `freedom_patches` and use `sane_patch`?

Reply:
We could probably use Sane Patch indeed. Moving this to freedom patches, I don’t know as freedom patches are all required by an initializer but this patch seems to be only required by the Unicorn conf file _if_ the `LOGSTASH_UNICORN_URI` env var is set.